app/controllers/transcriptions_controller.rb

```diff
@@ -25,10 +25,37 @@ def update
       authorize @transcription
     end
 
+    status_has_changed = status_has_changed(update_attrs)
```

Comment:
Definitely like the predicate idea. But I think I would need this line anyways, since I need to compare the status value on the `updated_attrs` to the transcription status _before_ the update is applied. Yet we need to generate the transcription files after the update, since we want the most recent transcription data to be reflected in the generated files. 

But I feel like there might still be a way to do this in a cleaner way...

Reply:
See comment directly below: https://github.com/zooniverse/tove/pull/101#discussion_r379650554

You can conditionally do the `update_transcription_exports` if `status_has_changed?`